app/src/main/java/org/literacyapp/authentication/RecognitionThread.java

```diff
@@ -80,17 +83,20 @@ private synchronized Student getMostSimilarStudentIfInThreshold(Mat featureVecto
         List<StudentImageCollectionEvent> studentImageCollectionEvents = studentImageCollectionEventDao.queryBuilder().where(StudentImageCollectionEventDao.Properties.MeanFeatureVector.isNotNull()).list();
```

Comment:
Possible to fetch only events where the Student (uniqueId) is not the same as the current Student?